lib/DueDatePicker/DueDateHeader.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import cx from 'classnames';
+import DueDateTime from './DueDateTime';
+import DropdownMenu from '../DropdownMenu';
+import Icon from '../Icon';
+
+const DueDateHeader = props => {
+  if (props.dueDate) {
+    const date = moment(props.dueDate);
+
+    const day = date.calendar(null, {
+      sameDay: '[Today at]',
+      nextDay: '[Tomorrow at]',
+      nextWeek: 'dddd [at]',
+      lastDay: '[Yesterday at]',
+      lastWeek: '[Last] dddd [at]',
+      sameElse: 'MMMM Do YYYY [at]'
+    });
+
+    const time = props.dueTime.format('LT');
+
+    const classes = cx('duedate__header', {
+      'duedate__header--overdue': date < moment().set({ hour: 0, minute: 0 })
```

Comment:
If you just want to set the text colour red there is a help class for that. Check our the helper styles 👍